adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMSaveAnyArgs.scala

```diff
@@ -0,0 +1,55 @@
+/**
+ * Licensed to Big Data Genomics (BDG) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The BDG licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bdgenomics.adam.rdd
+
+import org.bdgenomics.utils.cli.SaveArgs
+
+/**
+ * Argument configuration for saving any output format.
+ */
+trait ADAMSaveAnyArgs extends SaveArgs {
+
+  /**
+   * If true and saving as FASTQ, we will sort by read name.
+   */
+  var sortFastqOutput: Boolean
```

Comment:
is now a good time to rid the world of this?

Reply:
Don't think so.

Reply:
ok